cognitive/src/main/scala/com/microsoft/azure/synapse/ml/cognitive/OpenAI.scala

```diff
@@ -34,7 +34,7 @@ trait HasSetServiceName extends Wrappable with HasURL {
 
 trait HasPrompt extends HasServiceParams {
   val prompt: ServiceParam[String] = new ServiceParam[String](
-    this, "prompt", "The text to complete", isRequired = true)
+    this, "prompt", "The text to complete", isRequired = false)
```

Comment:
>false [](http://example.com/codeflow?start=57&length=5)

I'm assuming the orElse(throw...) near the bottom of this file will handle the case where none of the prompt formats are specified.